lndclient/lightning_client.go

```diff
@@ -61,6 +65,29 @@ type Info struct {
 	Uris           []string
 }
 
+// ChannelInfo stores unpacked per-channel info.
+type ChannelInfo struct {
+	// Active indecates whether the channel is active.
+	Active bool
+
+	// ChannelID holds the unique channel ID for the channel. The first 3 bytes
+	// are the block height, the next 3 the index within the block, and the last
+	// 2 bytes are the /output index for the channel.
+	ChannelID uint64
+
+	// PubKeyBytes is the raw bytes of the public key of the remote node.
+	PubKeyBytes route.Vertex
+
+	// Capacity is the total amount of funds held in this channel.
+	Capacity int64
```

Comment:
Can be type `btcutil.Amount`, just as the balances below

Reply:
indeed, that is much better